messaging/src/main/java/org/axonframework/commandhandling/gateway/CommandGateway.java

```diff
@@ -19,6 +19,7 @@
 import org.axonframework.commandhandling.CommandCallback;
 import org.axonframework.commandhandling.CommandExecutionException;
 import org.axonframework.commandhandling.CommandMessage;
+import org.axonframework.commandhandling.distributed.CommandDispatchException;
```

Comment:
I feel a little uncomfortable about introducing additional import from the distributed package. If you think we should not "pollute" `CommandGateway` with the distributed concepts, I'm ready to revert it.

Reply:
Honestly, I am on two legs with this one. 
From one point of view, the `CommandGateway` by no means specify you will distribute your application.

On the other end, this is "pre-Axon 3 speak", as back then the non-distribute route was taken most.
Nowadays, we virtually _always_ use a distributed version of the `CommandBus`, regardless of whether that's the `AxonServerCommandBus` or the `DistributedCommandBus`.

I would more so like to have the JavaDoc show we lean towards the distributed spectrum nowadays rather than not.
So, let's keep it. I'll put it up as a discussion point internally whether we'd want to change the location of the `CommandDispatchException` or to adjust this JavaDoc.

On any note, thanks for voicing your concern on the matter @tsobe; it's much appreciated.